src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,179 @@
+                            String err_msg = "☹ OOPS!!! The description or date of a deadline is wrong, plase key in the" +
+                                    "date in the format of yyyy-mm-dd, eg. 2001-02-10\n"
+                                    + "You may key in: deadline hw1 /2001-02-10, Duke.Duke will record your deadline for hw1 as" +
+                                    "2001-02-10";
+                            throw new DukeException(err_msg);
+                        }
+                        continue;
+                    case "EVENT":
+                        String[] event_str_arr = input.split(" /");
+                        if (len <= 9 || event_str_arr.length <= 2 || event_str_arr[0].length() < 7) {
+                            String err_msg = "☹ OOPS!!! The description or date of a event cannot be empty";
+                            throw new DukeException(err_msg);
+                        }
+                        try {
+                            System.out.println(Arrays.toString(event_str_arr));
+                            LocalDate from = LocalDate.parse(event_str_arr[1]);
+                            LocalDate to = LocalDate.parse(event_str_arr[2]);
+                            if (from.isAfter(to)){
+                                String err_msg = "☹ OOPS!!! Your time range is from a date to another date that is earlier" +
+                                        "than the former. Please key in a valid time range";
+                                throw new DukeException(err_msg);
+                            }
+                            Event event = new Event(event_str_arr[0].substring(6), from, to, 0);
+                            ArrayList arraylist = storage.load();
+                            arraylist.add(event);
+                            System.out.println("Got it. I've added this task");
+                            storage.update_data(arraylist);
+
+                            System.out.println("added: " + event);
+                            System.out.println("Now you have " + arraylist.size() + " tasks in the list");
+                        } catch (DateTimeParseException e) {
+                            String err_msg = "☹ OOPS!!! The description or date for the event is wrong, plase key in the" +
+                                    "date in the format of yyyy-mm-dd, eg. 2001-02-10\n"
+                                    + "You may key in: event hw1 /2001-02-10 /2001-02-12, Duke.Duke will record your event hw1 as" +
+                                    "from 2001-02-10 to 2001-02-12";
+                            throw new DukeException(err_msg);
+                        }
+                        continue;
+                    case "ERROR":
+                        String err_msg = "☹ OOPS!!! I'm sorry, but I don't know what that means";
+                        throw new DukeException(err_msg);
+                    }
+                }catch (DukeException e){
```

Comment:
Would strongly recommend splitting these into separate classes and packages for better extensibility. The current codebase looks very difficult to modify. Moreover, I'd suggest using enums.